util/grpc/src/grpcio_extensions.rs

```diff
@@ -86,16 +86,30 @@ impl ConnectionUriGrpcioServer for ServerBuilder {
                 .tls_key_path()
                 .expect("Uri must have tls-key in when using TLS");
 
-            let reloader = ServerCertReloader::new(&tls_chain_path, &tls_key_path, logger)
+            let reloader = ServerCertReloader::new(&tls_chain_path, &tls_key_path, logger.clone())
                 .expect("Failed creating ServerCertReloader");
 
+            log::debug!(
+                logger,
+                "Binding secure gRPC server to {}:{}",
+                uri.host(),
+                uri.port(),
+            );
+
             self.bind_with_fetcher(
                 uri.host(),
                 uri.port(),
                 Box::new(reloader),
                 CertificateRequestType::DontRequestClientCertificate,
             )
         } else {
+            log::warn!(
```

Comment:
Just double checking that this isn't left over debug code and that the intent is to now warn on a non TLS connections

Reply:
I'm keeping it to mirror the logging when connecting to an insecure channel.